CrewChiefV4/Events/Fuel.cs

```diff
@@ -530,6 +573,22 @@ public override void respond(String voiceMessage)
             {
                 reportFuelStatus();
             }
+            else if (SpeechRecogniser.ResultContains(voiceMessage, SpeechRecogniser.CALCULATE_FUEL_FOR))
+            {
+                int units = 0;
+                foreach (KeyValuePair<String, int> entry in SpeechRecogniser.numberToNumber)
+                {
+                    if (voiceMessage.Contains(" " + entry.Key))
```

Comment:
do we need voiceMessage.Contains(" " + entry.Key + " ")
here?

If the request is "how many litres for six laps" this will match " six" and "sixty", but not " sixty ". So it would depend on the order in which the SpeechRecogniser.numberToNumber Dictionary is iterated, which isn't (I think...) determinate